webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -42,22 +133,31 @@ function GithubItems(props) {
             >
                 <div>
                     <strong>
-                        {title}
+                        {title}{status}
                     </strong>
                 </div>
+                <div>
+                    {number} <span className='light'>{repoName}</span>
```

Comment:
Could we add here parenthesis to the repo name as the original design below?

![66054464-05b5b300-e502-11e9-8f70-cac639ef5322](https://user-images.githubusercontent.com/936315/72629538-874d1780-391e-11ea-9a1e-cfa2cede505f.png)